specs/shell.nix

```diff
@@ -1,6 +1,6 @@
 let
   # adding the haskell environment for running lhs2tex
-  pkgs = import (import ./../nixpkgs.nix) { config = import ./../config.nix; };
+  pkgs = import (import ./../nixpkgs.nix) { config = import ./../default.nix; };
```

Comment:
I'm not sure if we even need `nixpkgs.nix` but this is not required to be the bleeding edge anyway, so it's good.